11/alpine/hotspot/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM adoptopenjdk/openjdk11:jdk-11.0.14.1_1-alpine AS jre-build
+FROM adoptopenjdk/openjdk11:jdk-11.0.15_10-alpine AS jre-build
```

This seems to work, at least on my Intel architecture machiine
```suggestion
FROM eclipse-temurin:11.0.15_10-jdk-alpine AS jre-build
```